src/components/ReportActionItem/ReportPreview.js

```diff
@@ -181,7 +184,7 @@ function ReportPreview(props) {
                 <View style={[styles.reportPreviewBox, props.isHovered || isScanning || props.isWhisper ? styles.reportPreviewBoxHoverBorder : undefined]}>
                     {hasReceipts && (
                         <ReportActionItemImages
-                            images={_.map(lastThreeTransactionsWithReceipts, ({receipt, filename}) => ReceiptUtils.getThumbnailAndImageURIs(receipt.source, filename || ''))}
+                            images={lastThreeReceipts}
```

Comment:
This shouldn't be necessary since `lastThreeTranasactionsWithReceipts` already has a receipt check [here](https://github.com/Expensify/App/blob/arosiclair-distance-receipt/src/libs/ReportUtils.js#L3467).